cmd/manager/main.go

```diff
@@ -137,6 +139,16 @@ func main() {
 	var generatedClient kubernetes.Interface = kubernetes.NewForConfigOrDie(mgr.GetConfig())
 	propagator.Initialize(cfg, &generatedClient)
 
+	cache := mgr.GetCache()
```

Comment:
Hi Gus @gparvin , I did not understand where/what  this index function is used for.
Could you pls add a comment above just for one someone trying to understand the code later on.

everything else is good. thnx

Reply:
Good point.  Thanks

Reply:
@ckandag I just added a comment.  Can you look and see if it helps?